### DISCUSSION TITLE
G6 sensor is constantly loosing bluetooth pairing

### DISCUSSION THREAD

**Discussion by AidasK** [NightscoutFoundation/xDrip#2860] [General]
g6 sensor looses bluetooth pairing every 1-24hours and I have to click "pair" button again. I have used g6 with the same phone for more than 6months now without any problems. Troubles started 3days ago when I have inserted new sensor. Sensor was not pairing at all with bluetooth with app version 2022-12-??. Once app was updated to latest  2023.05.26, it started working! Though connection is not stable, bluetooth bond is constantly dropping. 

Phone: Google Pixel 6
OS: 13, May 5

Logs:
![photo_2023-05-29 15 16 09 (1)](https://github.com/NightscoutFoundation/xDrip/assets/2088484/aac94a25-d15f-44a4-9002-135af5f198df)
![photo_2023-05-29 15 16 09](https://github.com/NightscoutFoundation/xDrip/assets/2088484/24b20b6b-fc0c-4eea-bf72-a025f4de0c54)

Settings:
![photo_2023-05-29 15 16 07](https://github.com/NightscoutFoundation/xDrip/assets/2088484/5091af30-e6df-4aa9-8006-d1ae11767e9c)
![photo_2023-05-29 15 16 00](https://github.com/NightscoutFoundation/xDrip/assets/2088484/f8c3d31a-0239-4d4c-9e66-a7b289783b4e)

**Comment by Navid200** [comment]
Would you please make the following two changes?

Enable "Special pairing workaround".
Enable "Authenticate before each read".

**Reply by AidasK** [reply]
Thanks, after enabling these and waiting for 1 minute I received a pair request once again. But following reads seemed to be ok so let's see how it goes.
